CommonKit/Sources/CommonKit/Services/KeychainStore.swift

```diff
@@ -47,35 +58,89 @@ public final class KeychainStore: SecuredStore {
         try? KeychainStore.keychain.removeAll()
         NotificationCenter.default.post(name: Notification.Name.SecuredStore.securedStorePurged, object: self)
     }
+}
+
+private extension KeychainStore {
+    func configure() {
+        guard let privateKey = secureStorage.getPrivateKey(),
+              let publicKey = secureStorage.getPublicKey(privateKey: privateKey)
+        else { return }
+                
+        if let savedKey = getData(for: keychainStoreIdAlias) {
+            let decryptedData = secureStorage.decrypt(
+                data: savedKey,
+                privateKey: privateKey
+            ) ?? Data()
+            
+            keychainPassword = String(data: decryptedData, encoding: .utf8)
+            return
+        }
+        
+        let randomID = String.random(length: 32)
```

Comment:
https://medium.com/@peterdestasio/a-comprehensive-guide-to-data-encryption-in-swift-for-ios-development-b323b69c6556

Let's use best practices on how generate a random key:
```
// Generate a random encryption key
let key = SymmetricKey(size: .bits256)
```